### PR TITLE
COMMONS-567 : Notifications fall back to English for languages with country code

### PR DESCRIPTION
**Pull request by ahamdi**
Change Locale.getLanguage() by Locale.toString() to preserve the country in the language code

### CODE REVIEW

File: commons-api/src/main/java/org/exoplatform/commons/api/notification/plugin/NotificationPluginUtils.java [hunk 1]
```diff
@@ -148,7 +148,7 @@ public static String getLanguage(String userId) {
     String lang = null;
     if(localePolicy != null) {
       Locale locale = localePolicy.determineLocale(localeCtx);
-      lang = locale.getLanguage();
+      lang = locale.toString();
```
Comment (thomasdelhomenie) [inline commons-api/src/main/java/org/exoplatform/commons/api/notification/plugin/NotificationPluginUtils.java]: Please update org.exoplatform.commons.notification.NotificationUtilsTest to cover this case.

Comment (ahamdi) [reply commons-api/src/main/java/org/exoplatform/commons/api/notification/plugin/NotificationPluginUtils.java]: DONE

Comment (thomasdelhomenie) [reply commons-api/src/main/java/org/exoplatform/commons/api/notification/plugin/NotificationPluginUtils.java]: Actually org.exoplatform.commons.notification.NotificationUtilsTest is the class test for org.exoplatform.commons.notification.NotificationUtils, my bad. Can you move your test to a new test class for org.exoplatform.commons.api.notification.plugin.NotificationPluginUtils please ?

Comment (ahamdi) [reply commons-api/src/main/java/org/exoplatform/commons/api/notification/plugin/NotificationPluginUtils.java]: @thomasdelhomenie Where should I create the new Test class ? knowing that the org.exoplatform.commons.api.notification.plugin.NotificationPluginUtils is under the module commons-api which does not have test classes (which I find logic)

Comment (thomasdelhomenie) [reply commons-api/src/main/java/org/exoplatform/commons/api/notification/plugin/NotificationPluginUtils.java]: The test class must be in the same module and the same package, so it should be in commons-api/src/test/java/org/exoplatform/commons/api/notification/plugin.

**Pull request merged**